cmd/beacon-chain/flags/config.go

```diff
@@ -56,6 +57,11 @@ func ConfigureGlobalFlags(ctx *cli.Context) error {
 		cfg.SubscribeAllDataSubnets = true
 	}
 
+	if ctx.Bool(SemiSupernode.Name) {
+		log.Warning("Operating in semi-supernode mode (64 data columns)")
```

Comment:
Not sure you want to mention explicitly `64` here since it could change?


```suggestion
		log.Warning("Operating in semi-supernode mode (retain 50% of data columns)")
```